sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSinkSuite.scala

```diff
@@ -33,6 +33,19 @@ import org.apache.spark.util.Utils
 class FileStreamSinkSuite extends StreamTest {
   import testImplicits._
 
+  override def beforeAll(): Unit = {
```

Comment:
nit: a simpler way to fix this
```
override val conf = new SQLConf().copy(SQLConf.ORC_IMPLEMENTATION -> "native")
```

Reply:
Hi, @cloud-fan . 
I tested it, but that doesn't work in this `FileStreamSinkSuite`.